### PR TITLE
Add Omniglot Dataset

### PR DESCRIPTION
**Pull request by christiangnrd**
All local tests but that unrelated one I reported in #151 are passing.

### CODE REVIEW

File: docs/src/datasets/vision.md [hunk 1]
```diff
@@ -26,4 +26,5 @@ EMNIST
 FashionMNIST
 MNIST
 SVHN2
+Omniglot
```
Comment (CarloLucibello) [inline docs/src/datasets/vision.md:29]: let's keep an alphabetic ordering here

File: src/MLDatasets.jl [hunk 2]
```diff
@@ -150,6 +152,7 @@ function __init__()
     __init__fashionmnist()
     __init__mnist()
     __init__svhn2()
+    __init__omniglot()
```
Comment (CarloLucibello) [inline src/MLDatasets.jl:155]: also here alphavetic ordering

File: src/datasets/vision/omniglot.jl [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+The Omniglot data set is designed for developing more human-like learning
+algorithms. It contains 1623 different handwritten characters from 50 different
+alphabets. Each of the 1623 characters was drawn online via Amazon's
+Mechanical Turk by 20 different people. Each image is paired with stroke data, a
+sequences of [x,y,t] coordinates with time (t) in milliseconds.
+
+# Arguments
+
+$ARGUMENTS_SUPERVISED_ARRAY
+- `split`: selects the data partition. Can take the values `:train`, `:test`, `:small1`, or `:small2data`.
```
Comment (CarloLucibello) [inline src/datasets/vision/omniglot.jl:51]:
```suggestion
- `split`: selects the data partition. Can take the values `:train`, `:test`, `:small1`, or `:small2`. 
```

File: test/runtests.jl [hunk 4]
```diff
@@ -26,6 +26,7 @@ no_ci_dataset_tests = [
     "datasets/vision/cifar100.jl",
     "datasets/vision/emnist.jl",
     "datasets/vision/svhn2.jl",
+    "datasets/vision/omniglot.jl"
```
Comment (CarloLucibello) [inline test/runtests.jl:29]: alphabetic ordering

File: docs/src/datasets/vision.md (hunk 1 above)
Comment (christiangnrd) [reply docs/src/datasets/vision.md:28]: Changed

File: src/MLDatasets.jl (hunk 2 above)
Comment (christiangnrd) [reply src/MLDatasets.jl:154]: Changed. I noticed that some sections don't have alphabetical ordering. Would it be worth opening a new pull request with these consistency fixes?

Comment (CarloLucibello) [reply src/MLDatasets.jl:154]: yes, any help in cleaning up the docs or the code is well accepted

**Pull request merged**